zookeeper-docs/src/documentation/content/xdocs/zookeeperAdmin.xml

```diff
@@ -265,18 +266,25 @@ server.3=zoo3:2888:3888</programlisting>
           role="bold">server.id=host:port:port</emphasis>. The parameters <emphasis
           role="bold">host</emphasis> and <emphasis
           role="bold">port</emphasis> are straightforward. You attribute the
-          server id to each machine by creating a file named
+          server id to each machine by setting <emphasis role="bold">server.id</emphasis>
+          to a unique integer for each zookeeper server.To keep backwards compatibility,
+          you can still creat a file named
           <filename>myid</filename>, one for each server, which resides in
           that server's data directory, as specified by the configuration file
-          parameter <emphasis role="bold">dataDir</emphasis>.</para></listitem>
+          parameter <emphasis role="bold">dataDir</emphasis>.If the unique id is both set in the
+          server.id of zoo.cfg and myid file,the server.id has the priority
```

Comment:
i think we should return an error in this case. this will make debugging problems that result from this situation hard.